File: VB/SolutionWithCustomWindow.Module.Win/Controllers/LinkViewController.vb
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.XtraLayout
Imports System.Windows.Forms

Namespace SolutionWithCustomWindow.Module.Win.Controllers
    Partial Public Class LinkViewController
        Inherits ViewController

        Public Sub New()
            Dim showWindowAction As New SimpleAction(Me, "Show Window", DevExpress.Persistent.Base.PredefinedCategory.View)
            showWindowAction.ImageName = "ModelEditor_Views"
            AddHandler showWindowAction.Execute, AddressOf showWindowAction_Execute
        End Sub
        Private Sub showWindowAction_Execute(ByVal sender As Object, ByVal e As SimpleActionExecuteEventArgs)
            Dim form As New NonXAFForm()
            form.Text = "Form with the XAF ListView"

            Dim layoutControl As New LayoutControl()
            layoutControl.Dock = System.Windows.Forms.DockStyle.Fill
            form.Controls.Add(layoutControl)

            Dim item1 As LayoutControlItem = layoutControl.Root.AddItem()
            Dim textBox1 As New TextBox()
            item1.Text = "Company"
            item1.Control = textBox1

            Dim listView As DevExpress.ExpressApp.View = Application.CreateListView(Application.CreateObjectSpace(), GetType(Person), False)
            listView.CreateControls()
            Dim item2 As LayoutControlItem = layoutControl.Root.AddItem()
            item2.Text = "Persons"
            item2.Control = DirectCast(listView.Control, Control)

            form.ShowDialog()
        End Sub
    End Class
End Namespace

File: CS/SolutionWithCustomWindow.Module.Win/Controllers/LinkViewController.cs
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.Persistent.BaseImpl;
using DevExpress.XtraLayout;
using System.Windows.Forms;

namespace SolutionWithCustomWindow.Module.Win.Controllers {
    public partial class LinkViewController : ViewController {
        public LinkViewController() {
            SimpleAction showWindowAction = new SimpleAction(this, "Show Window", DevExpress.Persistent.Base.PredefinedCategory.View);
            showWindowAction.ImageName = "ModelEditor_Views";
            showWindowAction.Execute += new SimpleActionExecuteEventHandler(showWindowAction_Execute);
        }
        void showWindowAction_Execute(object sender, SimpleActionExecuteEventArgs e) {
            NonXAFForm form = new NonXAFForm();
            form.Text = "Form with the XAF ListView";
  
            LayoutControl layoutControl = new LayoutControl();
            layoutControl.Dock = System.Windows.Forms.DockStyle.Fill;
            form.Controls.Add(layoutControl);

            LayoutControlItem item1 = layoutControl.Root.AddItem();
            TextBox textBox1 = new TextBox();
            item1.Text = "Company";
            item1.Control = textBox1;

            DevExpress.ExpressApp.View listView = Application.CreateListView(Application.CreateObjectSpace(typeof(Person)), typeof(Person), false);
            listView.CreateControls();
            LayoutControlItem item2 = layoutControl.Root.AddItem();
            item2.Text = "Persons";
            item2.Control = (Control)listView.Control;

            form.ShowDialog();
        }
    }
}
